src/utils/InputSystem/index.ts

```diff
@@ -0,0 +1,151 @@
+import { InputSystem } from 'utils/InputSystem/InputSystem'
+import { store } from 'store/store'
+import { gamepadSlice } from 'store/modules/gamepad/reducer'
+import { buttons, sticks } from 'utils/InputSystem/mappings'
+import { rosClient } from 'utils/ros/rosClient'
+import { Action } from 'utils/InputSystem/@types'
+import { TopicOptions } from 'utils/ros/roslib-ts-client/@types'
+import { ITwistMsg, IJoyMsg } from 'utils/ros/rosMsgs.types'
+import { Vector3 } from 'utils/math/types'
+
+export const cmdVelTopic: TopicOptions = {
+  name: '/cmd_vel',
+  messageType: 'geometry_msgs/Twist',
+}
+
+export const joyTopic: TopicOptions = {
+  name: '/joy',
+  messageType: 'sensor_msgs/Joy',
+}
+
+export const spaceMouseTopic: TopicOptions = {
+  name: '/spacenav/twist',
+  messageType: 'geometry_msgs/Twist',
+}
+
+let joySeqId = 0
+
+export const mapGamepadToJoy = (gamepad: Gamepad): IJoyMsg => {
+  const d = new Date()
+  const seconds = Math.round(d.getTime() / 1000)
+
```

Comment:
Nitpick : Unnecessary newline ?

Reply:
I split it because it's declaring variables for 2 different concepts. The first 2 are for the timings the other are for deadzone/input handling

Reply:
Right I wasn't sure, we should leave it then.